docs/mockwbemserver.rst

```diff
@@ -10,26 +10,26 @@ Mock WBEM server
 Overview
 --------
 
-The pywbem package contains the ``pywbem_mock`` subpackage which provides mock
-WBEM server support that enables using the pywbem library without a WBEM
-server. This subpackage is useful for testing the pywbem library itself as well
+The 'pywbem_mock' module of pywbem provides mock WBEM server support that
```

Comment:
Change "mock WBEM server support" to "a mock WBEM server"

Reply:
DONE